src/sparseml/keras/utils/exporter.py

```diff
@@ -74,6 +75,16 @@ def export_onnx(
         if keras2onnx_import_error is not None:
             raise keras2onnx_import_error
 
+        if warn_on_tf_support:
+            import tensorflow
+            v = tensorflow.__version__
+            if v >= "2.3.0":
+                raise ValueError(
```

Comment:
nit: do we want to warn or raise an exception here?

Reply:
The flows currently don't work for most models in these versions, so preference is to raise an error to make sure the user sees that first and their first course of action to fix is ideally downgrading the tf version. We want to make sure they can consciously override it, though, to have it go through.

@natuan can we change the name from warn to raise to better signify what will happen?

Additionally in the error message let's change it to something along the lines of:
"Tensorflow version {} is greater than the currently supported version for keras2onnx. Please downgrade the Tensorflow version <2.3.0 or set raise_on_tf_support to False to continue."